pkgs/servers/maddy/default.nix

```
@@ -1,21 +1,30 @@
-{ lib, buildGoModule, fetchFromGitHub, coreutils, installShellFiles, scdoc, nixosTests }:
+{ lib, buildGoModule, fetchFromGitHub, coreutils, installShellFiles, scdoc, nixosTests, fetchurl }:
 
 buildGoModule rec {
   pname = "maddy";
-  version = "0.5.4";
+  version = "0.6.1";
 
   src = fetchFromGitHub {
     owner = "foxcpp";
     repo = "maddy";
     rev = "v${version}";
-    sha256 = "sha256-FWoPAb/aHaQLxT+UUUoViCmLvauVuAzUyOmRNB8F72U=";
+    sha256 = "sha256-CC+Aqo2fq46/TM9OpktlfpgXUvm8w8vHAU/DG7p/Vow=";
   };
 
-  vendorSha256 = "sha256-rcHboPfs2mWg3sgsLmN1IPoppmuDcsx0bQICp6EzYsQ=";
+
+  patches = [
+    # fixes user account creation: https://github.com/foxcpp/maddy/pull/507
+    (fetchurl {
+      url = "https://github.com/NickCao/maddy/commit/15163e75202dc2287733e75f1fc9c7e235d53184.patch";
```

please use the merged commit
```suggestion
      url = "https://github.com/foxcpp/maddy/commit/78966adb08a9443a3de724a59b48d6022b0702bb.patch";
```
besides that, LGTM

---

Or wait a little more to see if @foxcpp would like to cut a 0.6.2 release? The 0.6.x things was a bit messy.

---

Will definitely cut 0.6.2 as there are more issues.

---

I tagged 0.6.2 btw.